### PR TITLE
atlasaction: add context parameter to migrate/apply

### PR DESCRIPTION
**Pull request by dorav**

### CODE REVIEW

**Review by rotemtam — COMMENTED** [review]
Why do we need the context here ? Version is a global var anyways

File: cmd/atlas-action/main.go [hunk 1]
```diff
@@ -44,9 +44,20 @@ func main() {
...
+	_, err := fmt.Fprintln(app.Stdout, atlasaction.Version)
+	app.Exit(0)
+	return err
+}
+
 // RunAction is a command to run one of the Atlas GitHub Actions.
 type RunAction struct {
-	Action string `help:"Command to run" required:""`
+	Action  string      `help:"Command to run" required:""`
+	Version VersionFlag `help:"Prints the version and exits"`
```
Comment (a8m) [inline cmd/atlas-action/main.go:60]: Does this print the version on `atlas-action version`?

Comment (dorav) [reply cmd/atlas-action/main.go:60]: on `--version`

**Review by a8m — APPROVED** [review]
I prefer we stay consistent with the version command, or remove it completely.

**Pull request merged**